### ISSUE TITLE
Feature request: dictionaries

### ISSUE THREAD

**Issue by LilithHafner** [JuliaText/Languages.jl#37]
Hi folks, I'm looking for a dictionary of all or the most common 100,000 words in a language (english). Is this package a good place to put such a list? Or is it already here and I'm missing it?

**Comment by LilithHafner** [comment]
On unix for english we have `readlines("/usr/share/dict/words")`, but this package could provide something more general.

**Comment by aviks** [comment]
Yes, we could very well have this in this package. I'd be happy to review a PR for this.
